src/client.ts

```diff
@@ -19,7 +19,7 @@ import { BASE_URL_DEFAULT } from './constants'
  * Note: Typically you should use it's child class `FissionUser`
  *       which is accessable either directly or via `Fission.login`
  */
-export default class Fission {
+export class Fission {
```

Comment:
any reason why you got rid of `default` here?

this prevents someone from doing:
`import Fission from '@fission-suite/client`

Reply:
ah nvm you export it as `default` in `index` :ok_hand: